sphinx/application.py

```diff
@@ -1304,11 +1304,9 @@ def is_parallel_allowed(self, typ: str) -> bool:
             allowed = getattr(ext, attrname, None)
             if allowed is None:
                 logger.warning(message_not_declared, ext.name)
-                logger.warning(__('doing serial %s'), typ)
```

Comment:
Now, I don't know it is the case, but if suppressed warnings are teated as errors when enabling `-W`, then I would do the following:

- we should allow warnings to be treated as non-errors *if* they are suppressed *and* if a (new) configuration value specifies it. 
- Use `suppressed_warnings_errors = False` to indicate that suppressed warning are to be ignored but have a `True` value by default.

If this is not the case, namely, if suppressed warnings are never treated as errors *even with* `-W`, then no need for a new configuration value as above.